setup.sh

```diff
@@ -14,6 +14,8 @@
 # Usage:
 #     ./setup.sh
 
+# curl -fsSL https://ollama.com/install.sh | sh
+
```

Comment:
meh, I'm not using it right now. Maybe in the future I will acquire a beefy GPU and a better model, but letting it run on CPU for an 8B parameter model I was underwhelmed by performance (quality; speed was fine)